### PR TITLE
Issue 144 remove bo

### LINKED ISSUE #144: BusinessObject and TR041
Take TR041 into consideration for the class tree.

### PR DESCRIPTION
**Pull request by tormodv**
Fix #144 

1) Removed Business object
2)Moved Group, renamed it to Editorial Group. Now a subclass of Editorial object. Ref #176

### CODE REVIEW

File: ontology/EBUCore/ebucore.owl [hunk 1]
```diff
@@ -2486,7 +2473,7 @@ ec:hasUsageRights rdf:type owl:ObjectProperty ;
 ###  http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#hasVersion
 ec:hasVersion rdf:type owl:ObjectProperty ;
               owl:inverseOf ec:isVersionOf ;
-              rdfs:range ec:BusinessObject ;
+              rdfs:range ec:EditorialObject ;
```
Comment (alexandre-rouxel) [inline ontology/EBUCore/ebucore.owl:2476]: update the DC terms

File: ontology/EBUCore/ebucore.owl [hunk 2]
```diff
@@ -2605,7 +2592,7 @@ ec:isCharacter rdf:type owl:ObjectProperty ;
 ###  http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#isChildOf
 ec:isChildOf rdf:type owl:ObjectProperty ;
              owl:inverseOf ec:isParentOf ;
-             rdfs:range ec:BusinessObject ;
+             rdfs:range ec:Asset ;
              dcterms:description "To link a BusinessOject or Resource to a parent."@en ;
              rdfs:label "Parent"@en .
```
Comment (alexandre-rouxel) [inline ontology/EBUCore/ebucore.owl:2597]: update DC terms

File: ontology/EBUCore/ebucore.owl [hunk 3]
```diff
@@ -2803,7 +2790,7 @@ ec:isPreviousInSequence rdf:type owl:ObjectProperty ;
 ###  http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#isRatingRelatedToBusinessObject
 ec:isRatingRelatedToBusinessObject rdf:type owl:ObjectProperty ;
                                    rdfs:domain ec:Rating ;
-                                   rdfs:range ec:BusinessObject ;
+                                   rdfs:range ec:EditorialObject ;
                                    dcterms:description "To identify the BusinessObject associated with a Rating."@en ;
                                    rdfs:label "Rated business object"@en .
```
Comment (alexandre-rouxel) [inline ontology/EBUCore/ebucore.owl:2795]: update the DC terms

File: ontology/EBUCore/ebucore.owl (hunk 1 above)
Comment (tormodv) [reply ontology/EBUCore/ebucore.owl:2473]: Thanks. I forgot. Updated now

**Review by alexandre-rouxel — APPROVED** [review]
Thank you.

**Pull request merged**